docs/development.rst

```diff
@@ -398,6 +398,81 @@ Common resources that a test might need,
 like e.g. a running MSColab server or a QApplication instance for GUI tests,
 are collected in :mod:`tests.fixtures` in the form of pytest fixtures that can be requested as needed in tests.
 
+Testing Keyring Features
+=========================
+
+This document outlines step-by-step instructions for testing keyring features using the command line.
+
+Prerequisites
+-------------
+
+1. **Confirm the Default Keyring Backend**
+
+   Use the following command to list available keyring backends and check which one is currently in use:
+
+   .. code-block:: bash
+
+       keyring --list-backends
+
+Command-Line Commands for Keyring
+-----------------------------------
+
+1. **Set a Password**
+
+   Store a password for a specific service and user:
+
+   .. code-block:: bash
+
+       keyring set SERVICE_NAME USERNAME
```

Comment:
the examples maybe should destinguish the http_auth protection and a user password. 
When you use the` keyring set MSCOLAB_AUTH_{url} user@example.com` then it needs a change to
keyring set MSCOLAB_AUTH_{url}MSCOLAB_auth_user_name

Reply:
for the WMS Servers this username can differ for each server, an example is given in [config.py ](https://github.com/Open-MSS/MSS/blob/develop/mslib/utils/config.py#L220)

Reply:
mscolab has besides the common http_auth also user accounts and they are not described here.

These can become tested when you add a user with a password to your local running server.

Reply:
an example here would be

keyring set http://localhost:8083 myname@mydomain

Then when you started mscolab and use the myname@mydomain it fills in your saved credentials and you are logged in